packages/core/src/App/Containers/RealAccountSignup/personal-details.jsx

```diff
@@ -181,7 +185,10 @@ class PersonalDetails extends React.Component {
             '{{field_name}} is not in a proper format.',
         ];
 
-        const alt_messages = ['{{field_name}} is required', '{{field_name}} is not in a proper format.'];
+        const alt_messages = {
+            phone: ['{{field_name}} is required', '{{field_name}} is not in a proper format.'],
+            date_of_birth: ['{{field_name}} is required', 'You must be 18 years old and above.'],
```

Comment:
where did we get this text from?

Reply:
I got the text from copywriting team 😄

Reply:
@balakrishna-binary these messages need to be put through the `localize` function or component for the extractor to extract the string. For an example, please refer to this `amount_messages` implementation in `packages/p2p/src/components/my-ads/form-ads.jsx`.

Reply:
Sure 👍